components/places/src/storage/bookmarks.rs

```diff
@@ -889,6 +889,10 @@ impl<'de> Deserialize<'de> for BookmarkTreeNode {
     }
 }
 
+pub fn bookmarks_get_url_for_keyword(db: &PlacesDb, keyword: &str) -> Result<Option<Url>> {
+    Ok(None)
```

Comment:
@linacambridge I'm assuming you just never got to this?

@tinoism If you're interested in finishing this up, this function will need to be completed. (also check with Lina first!)

Reply:
Got it!

Reply:
We chatted a bit on Slack! 😄 Thanks for taking this over, @tinoism!